src/components/ThirdPartyIntegration.js

```diff
@@ -5,6 +5,14 @@ import { IntegrationsSdk } from "../sdk/integrations";
 import { PluginsSdk } from "../sdk/plugins";
 import { CaptiveRazorpay } from "./CaptiveRazorpay";
 import { Section } from "./Section";
+import { useLayoutEffect } from "@wordpress/element";
```

Comment:
@sumathi3399  Can we plz make use of `URLSearchParams` for changing query params values?